.github/workflows/linting.yml

```diff
@@ -63,16 +63,34 @@ jobs:
   markdown-link-check:
     name: Markdown Links (modified files)
```

Comment:
I guess we're not strictly checking only modified files in the current setup.  PRs that don't touch the CSV file still cause that file to be checked.

Reply:
Yes, you are right and I can fix that really easy, however don't you think we should check all the links every period of time?
The links could be broken even if the file has never been touched.

Reply:
Per discussion on Automation Sync, the idea is to check all files in the [Periodic workflow](https://github.com/submariner-io/submariner-operator/blob/master/.github/workflows/periodic.yml) and only modified files here.